sensor_msgs/Image data
---
sensor_msgs/Image mask
bool has_five_contours